pkg/doc/verifiable/credential_ldp_test.go

```diff
@@ -370,6 +370,11 @@ func TestExtraContextWithLDP(t *testing.T) {
 	err = vc.AddLinkedDataProof(ldpContext, jsonld.WithDocumentLoader(createTestJSONLDDocumentLoader()))
 	r.NoError(err)
 
+	// change VC and sign it again
+	vc.Status.ID = "https://example.edu/status/25"
+	err = vc.AddLinkedDataProof(ldpContext, jsonld.WithDocumentLoader(createTestJSONLDDocumentLoader()))
+	r.NoError(err)
```

Comment:
if you change the document, the first proof would fail. I'm not sure I understand this test?

Reply:
@troyronda yes but verify logic not working